packages/stratocacher-layer-dynamo/src/index.js

```diff
@@ -0,0 +1,100 @@
+import Q from "q";
+import {DynamoDB} from "aws-sdk";
+import {Layer} from "stratocacher";
+
+var client, awsConfig, tableName;
+
+class Client {
+	constructor(awsConfig) {
+		const ddbClient = new DynamoDB(awsConfig);
+		this.getItem = Q.nbind(ddbClient.getItem, ddbClient);
+		this.putItem = Q.nbind(ddbClient.putItem, ddbClient);
+	}
+}
+
+function getClient() {
+	if (!client) {
```

Comment:
This should be memoized by `cid`.

Reply:
I thought we want one client for all instances?